src/vos/tests/vts_mvcc.c

```diff
@@ -788,6 +788,14 @@ querymaxd_f(struct io_test_args *arg, struct tx_helper *txh, char *path,
 	uint64_t	 dkey_val = 0;
 	daos_key_t	 dkey = {&dkey_val, sizeof(dkey_val), 0};
 
+	/* Test section */
+	int i;
+	int test_len = 10;
+	int *test_ptr = (int *)malloc(sizeof(int)*test_len);
+	for(i = 0; i < test_len; i++) {
```

Comment:
(style)  Missing a blank line after declarations
(style)  space required before the open parenthesis '('